README.md

```diff
@@ -334,6 +334,24 @@ You can use watermarking by adding a new udf step in your metric:
     delayThreshold: 2 hours
 ```
 
+##### ToAvro
+Metorikku supports to_avro() method which turns a dataframe into Avro records and send these to kafka.
+The method requires the following parameters: table, schema.registry.url, schema.registry.topic, value.schema.name, value.schema.namespace, schema.registry.topic, and kafka.url.
+The method will auto-create a topic in Kafka if one doesn't exist; moreover, a subject will be created in the schema registry (if one doesn't already exist). The subject name will be: <schema.registry.topic>-<value.schema.namespace>.<value.schema_name>.
+<br/>You can use ToAvro by adding a new udf step in your metric:
+```yaml
+- dataFrameName: dataframe
+  classpath: com.yotpo.metorikku.code.steps.ToAvro
+  params:
+    table: my_table
```

Comment:
What exactly does table mean?
We pass the dataframe name. why we need table as well?

Reply:
Table is the table we will load and convert to Avro. Dataframe is the name of the dataframe we call createOrReplaceTempView on (with the converted table)

Reply:
right. sorry

Reply:
That is, we do ss.table("my_table")